code/game/objects/machinery/vending/marine_vending.dm

```diff
@@ -1190,7 +1188,7 @@
 	icon_state = "bloodvendor"
 	icon_vend = "bloodvendor-vend"
 	icon_deny = "bloodvendor-deny"
-	product_ads = "The best blood on the market!"
+	product_slogans = "The best blood on the market!;Totally came from an ethical source!;O plus is the universal donor, use it!; Prevent hypovolemic shock starting today!"
```

Comment:
O negative is universal donor. depending on if this gets maptext or not, might be more useful not to spell it out and just have "O- is the universal donor, use it!"? that way the suggested blood type people see in maptext matches exactly the blood type listed in the vendor.